06_proxmox_lvm_resize_disk_ssh_access_backups_25-aug-2022/README.md

```diff
@@ -1,54 +1,16 @@
-Демо на setup access via SSH
+## SSH access, Proxmox LVM resize disk, Proxmox backups ##
 
-### via ssh agent ###
+| Example | Details |
+|------|-------|
+| [Instruction 1](./lvm.md) | How to resize LVM on Proxmox |

```

Comment:
I do not like the title Instruction here. May be Part? @pgagarinov what do you think?